### ISSUE TITLE
PHB Ranking Criteria Data - Sprint 17

### ISSUE THREAD

**Issue by jaime-mckeown** [cityofaustin/atd-geospatial#555]
- Continue working on ranking script.
- Need to modify script for creating Street_Select_PHB as there are some highways getting selected that should not be.
- Will update census data if there is time as the 2018 ACS data became available in December.

**Comment by jaime-mckeown** [comment]
- Updated the Street_Select_PHB script to remove more streets that Andres sent over. Also submitted a help desk ticket to have some of the streets that we found with incorrect road classes assigned. They have been fixed, still need to update the script to remove those streets from the codeblock.
- Updated the household income data to reflect 2018 data.
- We changed population density back to population per the entire block group as the numbers were getting skewed when we were looking at block groups that went beyond the city limits.
- Processed the 2018 population data to get population density based on overall area of the block group. Emailed AMD about the ranking not really working. Andres sent over a new ranking for population density. Got that added to the script.
- Ranking script is complete. The criteria that we are not using at this time are included, but are calculating to 0 so the total number for ranking will be less than 100.
